### PR TITLE
doc: update docs with AIX related information

### PR DESCRIPTION
**Pull request by iWuzHere**
As discussed in https://github.com/libuv/libuv/issues/882

### CODE REVIEW

File: docs/src/fs.rst [hunk 1]
```diff
@@ -258,6 +258,10 @@ API
 
     Equivalent to :man:`utime(2)` and :man:`futime(2)` respectively.
 
+    .. note::
+      For AIX, this function only works for AIX 7.1 and newer. It can still be called on older
```
Comment (saghul) [inline docs/src/fs.rst]: AIX:

File: docs/src/fs.rst [hunk 2]
```diff
@@ -258,6 +258,10 @@ API
...
+      versions but will return UV_ENOSYS.
```
Comment (saghul) [inline docs/src/fs.rst]: surround the constant with backticks

Comment (saghul) [inline docs/src/fs.rst]: make that double backticks.

**Comment by saghul** [comment]
A couple of nits, LGTM otherwise.

**Comment by iWuzHere** [comment]
Updated accordingly!

File: docs/src/fs_event.rst [hunk 3]
```diff
@@ -8,6 +8,16 @@ FS Event handles allow the user to monitor a given path for changes, for example
 if the file was renamed or there was a generic change in it. This handle uses
 the best backend for the job on each platform.
 
+.. note::
+    For AIX, the non default IBM bos.ahafs package has to be installed.
```
Comment (saghul) [inline docs/src/fs_event.rst]: can we link to some documentation here? or I guess AIX people are usually familiar with how to do that?

File: docs/src/fs_event.rst [hunk 4]
```diff
@@ -8,6 +8,16 @@ FS Event handles allow the user to monitor a given path for changes, for example
...
+    The AIX Event Infrastructure file system (ahafs) has some limitations
+
+        - ahafs tracks monitoring per process and is not thread safe. A seperate process
```
Comment (cjihrig) [inline docs/src/fs_event.rst]: seperate -> separate

File: docs/src/fs_event.rst [hunk 5]
```diff
@@ -8,6 +8,16 @@ FS Event handles allow the user to monitor a given path for changes, for example
...
+          must be spawned for each monitor for the same event.
+        - Events for file modification (writing to a file) are not recieved if only the
```
Comment (cjihrig) [inline docs/src/fs_event.rst]: recieved -> received

**Comment by cjihrig** [comment]
LGTM once comments are addressed.

**Comment by iWuzHere** [comment]
Updated as per feedback!

File: docs/src/fs.rst (hunk 2 above)
Comment (saghul) [inline docs/src/fs.rst]: double backticks

**Comment by iWuzHere** [comment]
Updated yet again!

**Comment by saghul** [comment]
LGTM, go for it!

**Comment by iWuzHere** [comment]
Landed https://github.com/libuv/libuv/commit/71bd07e588942b95009fbe055d4ae8cafe6532fa

**Pull request closed without merging**